tests/cloud/HL_testing_input.json

```diff
@@ -0,0 +1,17 @@
+{
+    "license": "SEE LICENSE IN LICENSE FILE",
+    "model_name" : ["gpt2","TinyLlama/TinyLlama-1.1B-Chat-v1.0","Salesforce/codegen-350M-mono","wtang06/mpt-125m-c4"],
+    "num_cores" : [8,16],
+    "prompt" : ["My name is"],
+    "prompts_txt_file_path" : ["examples/prompts.txt"],
+    "aic_enable_depth_first" : [0,1],
+    "mos" : [-1,1],
+    "cache_dir" : [null],
+    "hf_token" : [null],
+    "batch_size" : [1,2,4],
+    "prompt_len" : [16,32,64,128],
+    "ctx_len" : [128],

```

Comment:
With prompt_len 128 and ctx_len also 128, generation length will be zero so no tokens will be genertated, should we change ctx_len to 256? @ochougul

Reply:
Good catch.
We can always keep
```
ctx_len=prompt_len*2
```

Reply:
We can chnage ctx_len to 256 or we can remove the prompt_len of 128,